library/core/src/time.rs

```diff
@@ -48,6 +48,17 @@ const MICROS_PER_SEC: u64 = 1_000_000;
 ///
 /// let ten_millis = Duration::from_millis(10);
 /// ```
+///
+/// # Formatting `Duration` values
+///
+/// `Duration` intentionally does not have a `Display` impl, as there are a
+/// variety of ways to format spans of time for human readability. `Duration`
+/// provides a `Debug` impl that shows the full precision of the value.
+///
+/// The `Debug` output uses the Unicode "µs" suffix for microseconds. If your
```

Comment:
Maybe we should say `non-ASCII` instead of `Unicode` here, since `"ms"` etc. are also Unicode. (Or just leave out the word entirely.)

Reply:
Done.